NDB.Covid19/NDB.Covid19.iOS/Info.plist

```diff
@@ -61,10 +61,10 @@
 	<key>UIAppFonts</key>
 	<array>
 		<string>Fonts/PTSans-Bold.ttf</string>
-		<string>Fonts/PTSans-Regular.ttf</string>
 		<string>Fonts/PTSans-BoldItalic.ttf</string>
-		<string>Fonts/PTSans-Italic.ttf</string>
 		<string>Fonts/PTSans-Caption-Regular.ttf</string>
+		<string>Fonts/PTSans-Italic.ttf</string>
+		<string>Fonts/PTSans-Regular.ttf</string>
```

Comment:
Alphabetically sorted? :) Good :)

Reply:
Yep, I re-added them when debugging this :)